### DISCUSSION TITLE
How can I set the ‘Most Upvoted’ tab as the default instead of the ‘My Feed’ tab?

### DISCUSSION THREAD

**Discussion by QuangNguyen1607** [dailydotdev/daily#1032] [Feature Requests]
How can I set the ‘Most Upvoted’ tab as the default instead of the ‘My Feed’ tab?

**Comment by idoshamun** [comment]
Hi @QuangNguyen1607,

Unfortunately, it is not possible at the moment. My Feed is the default tab.
